build-monitor-plugin/src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/viewmodel/JobView.java

```diff
@@ -196,6 +196,12 @@ public int hashCode() {
         return Objects.hashCode(job.hashCode());
     }
 
+    @Override
```

Comment:
JobView is never used as a key in a hash map, but you're right, let's keep Joshua Bloch happy ;-)